packages/custom-types/channel-ui.d.ts

```diff
@@ -12,14 +12,14 @@ export default interface ChannelData {
   templates?: TemplateBody[],
   messages?: Array<MessageChannel>,
   name: string,
-  participants: Array<Participant>

```

Comment:
Hmm, I don't like this class. I _think_ it should have a child member that stores an object of `ChannelTypes`.  Then we will still have access to the channel description (including participants), and collab-options if it's a `ChannelCollab`.